src/main/java/frc/robot/Robot.java

```diff
@@ -190,10 +193,14 @@ public void testPeriodic() {
 
   public boolean isPowerCellLoaded(){
     isPowerCellLoaded = distanceSensorDetector.isPowerCellLoaded();
-    System.out.println("sensing ball" + isPowerCellLoaded);
     return isPowerCellLoaded;
   }
 
+  public boolean isLimelightLockedOn(){
+    isLimelightLockedOn = limelight.hasValidTarget();
```

Comment:
This will be true when the limelight sees the target, but doesn't mean that we're aimed correctly.  What is the intent of this feature?  I'm assuming it's to show when we're aimed correctly.  If so, you'll want to do the same check as is done in the LimelightAimCommand.

Reply:
this feature is intended for letting us know when to press the aim button I thought, and then we press a button to start a command sequence with aiming, distance, and then shooting

Reply:
also, tested code, me and garret jumped up and down in front of the target, and it works

Reply:
Awesome.